Sources/Purchasing/OfferingsManager.swift

```diff
@@ -40,17 +42,16 @@ class OfferingsManager {
     func offerings(
         appUserID: String,
         fetchPolicy: FetchPolicy = .default,
+        fetchCurrent: Bool = false,
```

Comment:
I nearly confused the `fetchPolicy` parameter above with the `CacheFetchPolicy` 😅. We should eventually move to that, but I'm ok with a boolean parameter for now.

Reply:
Same, TBH 😅  We thought about adding a similar enum didn't want to over complicate it. Should be easy to switch to an enum later if we want!